### ISSUE TITLE
Support AMD Js

### ISSUE THREAD

**Issue by sonisourabh** [IFCjs/web-ifc-three#117]
Does `web-ifc-three` supports AMD using [requirejs](https://requirejs.org/) way of writing js? Do you have any example app in AMD?

**Comment by makc** [comment]
well if you open https://cdn.jsdelivr.net/npm/web-ifc-viewer@1.0.215/dist/index.js you will see that it's this:
```js
export * from './ifc-viewer-api';
export * from './base-types';
//# sourceMappingURL=index.js.map
```
so loading it in any way other than via `import` will fail with that error

**Comment by cvillagrasa** [comment]
Indeed... do you know if it has any straightforward solution? or would the library need to be rewritten to work as an AMD?

**Comment by makc** [comment]
you can try something like `rollup dist/index.js --file dist/index.amd.js --format amd` maybe

**Comment by cvillagrasa** [comment]
I already did, the error is the same regardless of where it's run. Unfortunately, I think that every import would need to be changed to be [relative to a baseUrl](https://requirejs.org/docs/api.html).
